esky/util.py

```diff
@@ -499,7 +499,7 @@ def really_rmtree(path):
         if not os.path.exists(path):
             shutil.rmtree(path)
         #  This is a little retry loop that catches troublesome errors.
-        for _ in xrange(10):
+        for _ in xrange(100):
```

Comment:
i quote "(b) add a loop to sleep-and-retry the rmtree call until it succeeds"
I actually didn't try an infinite loop.. it could work to fix the problem 100%...